pairing_bot.go

```diff
@@ -25,86 +28,84 @@ type PairingLogic struct {
 	adb   APIAuthDB
 	pdb   PairingsDB
 	revdb ReviewDB
-	ur    userRequest
-	un    userNotification
-	sm    streamMessage
 	rcapi RecurseAPI
+
+	zulip *zulip.Client
 }
 
 var randSrc = rand.New(rand.NewSource(time.Now().UnixNano()))
 
 func (pl *PairingLogic) handle(w http.ResponseWriter, r *http.Request) {
 	var err error
 
-	responder := json.NewEncoder(w)
```

Comment:
I inlined this for now, but I intend to extract a `func handleWebhook(r *http.Request) error`  with basically the whole rest of the function body, which will restore the readability :slightly_smiling_face: